dashboard/migrations/0016_adding_mysql_cache.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
```

Comment:
This migration file will need to be renamed up to 19
16 is used by #993
17 is used by #981
18 is going to be used by #995